Python/Product/VSInterpreters/PackageManager/PipRequirementsUtils.cs

```diff
@@ -14,10 +14,12 @@
 // See the Apache Version 2.0 License for specific language governing
 // permissions and limitations under the License.
 
+using Microsoft.PythonTools.Infrastructure;
 using System;
 using System.Collections.Generic;
 using System.Linq;
 using System.Text.RegularExpressions;
+using System.Threading.Tasks;
```

Comment:
System should come first. Please check "Place 'System' directives first when sorting usings" and re-sort.

![image](https://user-images.githubusercontent.com/1696845/55194347-a21f8700-5166-11e9-8806-57adf785ebb5.png)